container/src/LuigiContainer.svelte

```diff
@@ -151,6 +152,15 @@
       webcomponentService.thisComponent = thisComponent;
 
       const ctx = GenericHelperFunctions.resolveContext(context);
+
+      thisComponent.updateViewUrl = (viewUrl: string, internal?: object) => {
+        if (viewUrl?.length) {
+          const context = Object.assign(ctx, { viewUrl });
```

Comment:
```suggestion
          const context = GenericHelperFunctions.resolveContext(context);
```
Otherwise context object would be always the initial one during container instance lifetime, and therefore a potentially changed context would be reset to the initial one when update the viewUrl